lib/wolfssl/atca_wolfssl_interface.c

```diff
@@ -351,7 +351,7 @@ ATCA_STATUS atcac_pk_init(
     {
         if (!key_type)
         {
-            ctx->ptr = wc_ecc_key_new(NULL);
+            ctx->ptr = (ecc_key*)wc_ecc_key_new(NULL);
```

Comment:
What compiler/settings is throwing a warning here? ctx->ptr is of type void * so I wouldn't expect it to throw a warning much less be resolved by casting to a different type